cmd/core/main.go

```diff
@@ -183,12 +185,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	controllerArgs.SysDefinitionNamespace = sysDefinitionNamespace
```

Comment:
why not directly use the `controllerArgs.SysDefinitionNamespace` as the flag var?